lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,19 @@
-
 def palindrome_permutation?(string)
```

Comment:
This solution also will not work for all palindromes, although it does satisfy the given tests.

Think about how you can handle this with a hash.